tutorials/assets_pipeline/importing_3d_scenes/import_configuration.rst

```diff
@@ -395,6 +395,8 @@ The ``_post_import(scene: Node)`` function takes the imported scene as argument
 (the parameter is actually the root node of the scene). The scene that will
 finally be used **must** be returned (even if the scene can be entirely different).
 
+To use your script, locate the script in import tab's "Path" option under "Import Script" category.
```

Comment:
```suggestion
To use your script, locate the script in the import tab's "Path" option under the "Import Script" category.
```

Reply:
Oh, it's always THE articles. No matter how I try to remember check for missing articles, there'll be a few 🤦‍♂️